creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -696,14 +704,19 @@ async function secondarySyncFromPrimary({
   )
   const metricEndTimerFn = secondarySyncFromPrimaryMetric.startTimer()
 
-  const mode = forceResyncConfig?.forceResync ? 'force_resync' : 'default'
+  const mode = forceWipe
```

Comment:
lolll ok can we not do this nested ternary pls
hard to follow and feels very error prone

Reply:
thought you might not like that 😂 I'll make it more explicit